mason.sh

```diff
@@ -40,16 +40,18 @@ esac
 
 if [ ${MASON_PLATFORM} = 'osx' ]; then
     export MASON_HOST_ARG="--host=x86_64-apple-darwin"
-    export MASON_PLATFORM_VERSION=`xcrun --sdk macosx --show-sdk-version`
+    export MASON_PLATFORM_VERSION="all"
```

Comment:
Using `x86_64` causes a failure in one of the tests. I'll need someone to help me figure out why https://travis-ci.org/mapbox/mason/jobs/115412535.